File: angular/src/app/models/document.model.ts
import { DocumentStatus } from './enums/document-status.enum';
import { DocumentType } from './enums/document-type.enum';

export class Document {
    public documentId: string;
    public intakeFormId: string;
    public physicianId: string;
    public type: DocumentType;
    public status: DocumentStatus;
    public content: string;
    public signature: string;
    public createdOn: Date;
    public modifiedOn: Date;
}

File: dotnet/PhysiciansReach/PR.Models/DocumentModel.cs
using PR.Constants.Enums;
using System;

namespace PR.Models
{
    public class DocumentModel
    {
        public int DocumentId { get; set; }

        public int IntakeFormId { get; set; }

        public int? PhysicianId { get; set; }

        public int? SignatureId { get; set; }

        public DocumentType Type { get; set; }

        public DocumentStatus Status { get; set; }

        public byte[] Content { get; set; }

        public byte[] Signature { get; set; }

        public DateTime CreatedOn { get; set; }

        public DateTime ModifiedOn { get; set; }
    }
}
